Add an option to export data in a format compatible with Kuzzle fixtures (#82)
kuzzleio/kourou: README.md

  --as=as                  Impersonate a user
  --batch-size=batch-size  [default: 2000] Maximum batch size (see limits.documentsFetchCount config)
  --editor                 Open an editor (EDITOR env variable) to edit the query before sending

  --format=format          [default: JSONL] "jsonl or kuzzle - kuzzle will export in Kuzzle format usable for internal

>                            fixtures and jsonl will be usable to re import those data whith kourou

Reviewer: ```suggestion
                           fixtures and jsonl allows to import that data back with kourou
```

  --api-key=api-key        Kuzzle user api-key
  --as=as                  Impersonate a user
  --batch-size=batch-size  [default: 2000] Maximum batch size (see limits.documentsFetchCount config)

  --format=format          [default: jsonl] "jsonl or kuzzle - kuzzle will export in Kuzzle format usable for internal

>                            fixtures and jsonl will be usable to re import those data whith kourou

Reviewer: ```suggestion
                           fixtures and jsonl allows to import that data back with kourou
```
